gui/pages/kernel_options.go

```diff
@@ -0,0 +1,292 @@
+		}
+		box.PackStart(labelName, false, false, 0)
+
+		labelDesc, err := setLabel(desc, "list-label-desc", 0.0)
+		if err != nil {
+			return nil, err
+		}
+		box.PackStart(labelDesc, true, true, 0)
+
+		page.list.Add(box)
+	}
+
+	// addLabel: Tell users they can add kernel command lines below
+	addText := utils.Locale.Get("Add Extra Arguments:")
+
+	page.addLabel, err = setLabel(addText, "label-addLabel", 0.0)
+	if err != nil {
+		return nil, err
+	}
+	page.addLabel.SetMarginStart(common.StartEndMargin)
+	page.addLabel.SetHAlign(gtk.ALIGN_START)
+	page.box.PackStart(page.addLabel, false, false, 10)
+
+	// addEntry: Args to add to the kernel command line
+	page.addEntry, err = setEntry("addEntry")
+	if err != nil {
+		return nil, err
+	}
+	page.addEntry.SetMarginStart(common.StartEndMargin)
+	page.addEntry.SetMarginEnd(common.StartEndMargin)
+	page.addEntry.SetTooltipText(utils.Locale.Get(kernelArgsHelp))
+	page.box.PackStart(page.addEntry, false, false, 0)
+
+	// remLabel label
+	remText := utils.Locale.Get("Remove Arguments:")
+	page.remLabel, err = setLabel(remText, "label-remLabel", 0.0)
+	if err != nil {
+		return nil, err
+	}
+	page.remLabel.SetMarginStart(common.StartEndMargin)
+	page.remLabel.SetMarginEnd(common.StartEndMargin)
+	page.box.PackStart(page.remLabel, false, false, 10)
+
+	// remEntry: Args to remove to the kernel command line.
+	page.remEntry, err = setEntry("remEntry")
+	if err != nil {
+		return nil, err
+	}
+	page.remEntry.SetMarginStart(common.StartEndMargin)
+	page.remEntry.SetMarginEnd(common.StartEndMargin)
+	page.remEntry.SetTooltipText(utils.Locale.Get(kernelArgsHelp))
+	page.box.PackStart(page.remEntry, false, false, 0)
+
+	return page, nil
+}
+
+func (page *ConfigKernelPage) getKern() *kernel.Kernel {
+	if page.model.Kernel != nil {
+		return page.model.Kernel
+	} else {
+		// if model is empty, return kernel-native
+		for _, v := range page.data {
+			if v.Bundle == "kernel-native" {
+				return v
+			}
+		}
+	}
+
+	return nil
+}
+
+func (page *ConfigKernelPage) onRowActivated(box *gtk.ListBox, row *gtk.ListBoxRow) {
+	page.selected = page.data[row.GetIndex()]
+	page.controller.SetButtonState(ButtonConfirm, true)
+}
+
+// Select row in the box, activate it and scroll to it
+func (page *ConfigKernelPage) activateRow(index int) {
+	row := page.list.GetRowAtIndex(index)
+	page.list.SelectRow(row)
+	page.onRowActivated(page.list, row)
+	scrollToView(page.scroll, page.list, &row.Widget)
+}
+
+// IsRequired will return false as we have default values
+func (page *ConfigKernelPage) IsRequired() bool {
+	return false
+}
+
+// IsDone checks if all the steps are completed
+func (page *ConfigKernelPage) IsDone() bool {
+	//Always true because all steps are optional
+	return true
+}
+
+// GetID returns the ID for this page
+func (page *ConfigKernelPage) GetID() int {
+	return PageIDConfigKernel
+}
+
+// GetIcon returns the icon for this page
+func (page *ConfigKernelPage) GetIcon() string {
+	return "computer"
```

Comment:
```suggestion
	return "applications-engineering"
```

Reply:
Please update to a unique icon; I like this one best for now.

Reply:
Done.